packages/core/tests/test_assets_deploy.py

```diff
@@ -1,12 +1,61 @@
-from bag3d.core.assets.deploy.godzilla import (
+from bag3d.core.assets.deploy.servers import (
     compressed_export_nl,
     downloadable_godzilla,
+    downloadable_podzilla,
 )
 from pathlib import Path
-from fabric import Connection
 import pytest
 
 
+@pytest.mark.needs_tools
+def test_downloadable_podzilla(context, test_data_dir):
+    # Create deployment dir
+    export_dir = test_data_dir / "deployment" / "3DBAG" / "export_test_version"
+    export_dir.mkdir(parents=True, exist_ok=True)
+
+    # Create an empty file within the directory
+    empty_file = export_dir / "dummy.txt"
+    empty_file.touch()
+
+    # Create a mock metadata file
+    metadata_file = test_data_dir / "deployment" / "3DBAG" / "metadata.json"
+    metadata_file.touch()
+    metadata_file.write_text(
+        '{"identificationInfo": {"citation": {"edition": "test_version"}}}'
+    )
+    try:
+        # compress the export dir
+        res = compressed_export_nl(context, export_dir)
+
+        compressed_file = Path(res.metadata["path"].text)
+        assert compressed_file.exists()  # Check that the file was created
+
+        # Test the transfer to podzilla
+        res = downloadable_podzilla(
+            context,
+            compressed_file,
+            metadata_file,
+        )
+        assert (
+            res == f"{context.resources.godzilla_server.dir}/test_version"
```

Comment:
Why do you use the godzilla_server resource in the podzilla test, instead of the podzilla_server resource? Same in the lines below.